src/app/documents/scan-qr-code/scan-qr-code.component.scss

```diff
@@ -37,7 +37,7 @@
 .img-center {
     margin: 0;
     position: absolute;
-    top: 50%;
+    top: 380px;
```

Comment:
@shreyas1434shinde will it work for all the devices? have u tested on different device layouts

Reply:
@itsvick yes it will work to all devices. I have test it